backend/src/main/java/saci/domain/service/LevelService.java

```diff
@@ -0,0 +1,46 @@
+package saci.domain.service;
+
```

Comment:
Don't forget to do the clean up of everything not related to the ticket in the backend, as Vitor said.